cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -76,15 +77,18 @@ function ManagementCredentials(props: Partial<StepProps>) {
             azureCloud: azureState[STORE_SECTION_FORM].AZURE_ENVIRONMENT,
         };
         try {
+            setConnectionStatus(CONNECTION_STATUS.CONNECTING);
+            setMessage('Connecting to Azure');
             await AzureService.setAzureEndpoint(params);
-            setConnection(true);
+            setConnectionStatus(CONNECTION_STATUS.CONNECTED);
+            setMessage('Connected to Azure');
             retrieveRegions();
-        } catch (e) {
-            console.log('Error');
+        } catch (err: any) {
+            setMessage(`Unable to connect to Azure: ${err.body.message}`);
```

Comment:
on the error, I think (if I recall correctly) the connection status should be set to ERROR.